### DISCUSSION TITLE
Github 404 error 'This is not the webpage you are looking for'

### DISCUSSION THREAD

**Discussion by donprofaghatise** [community/community#27374] [Other Feature Feedback, Questions, & Ideas]
Appreciate any help. I got this Github link from a journal paper related to uncertainty quantification code. When I click on the link, I get a 404 error. Appreciate any help in resolving this. See the link below.

https://github.com/Xiaohu-Zheng/Deep-Polynomial-Chaos-Neural-Network-Method

**Comment by airtower-luna** [comment]
That is unfortunate, a 404 Not Found error for a repository usually means either the repository doesn't exist (maybe it has been deleted), or it's private and you don't have access. :confused: 

I think your best bet is to contact the user who used to own the repository or the author of the article, if possible.

**Reply by donprofaghatise** [reply]
Thank you so much for the feedback

**Reply by ImStven** [reply]
I am the owner of this link "https://github.com/ImStven/react-project" but I get that error and I already posted it publicly but I still get that error, can you please help me?

**Reply by XintongYang1010** [reply]
> I am the owner of this link "https://github.com/ImStven/react-project" but I get that error and I already posted it publicly but I still get that error, can you please help me?

I met the same problem, how did you solve it?

**Comment by radar-blip** [comment]
I think maybe its been renamed.

**Reply by airtower-luna** [reply]
Also possible, though in that case there'll usually be a redirect. :thinking:

**Reply by donprofaghatise** [reply]
Thanks so much for the suggestion/thoughts

**Comment by fernandounger** [comment]
The repository owner may have made it private or deleted it.

**Reply by donprofaghatise** [reply]
Thanks so much for the suggestion/thoughts

**Reply by connorballew-lab** [reply]
i dont think this works// ==UserScript==
// @name         Blooket Coin and Token Hacker
// @namespace    http://tampermonkey.net/
// @version      1.0
// @description  Add coins and tokens to your Blooket account
// @author       You
// @match        https://dashboard.blooket.com/market
// @grant        none
// @license      MIT
// @downloadURL https://update.greasyfork.org/scripts/516991/Blooket%20Coin%20and%20Token%20Hacker.user.js
// @updateURL https://update.greasyfork.org/scripts/516991/Blooket%20Coin%20and%20Token%20Hacker.meta.js
// ==/UserScript==

(function() {
    'use strict';

    // Function to add coins and tokens
    function addCoinsAndTokens() {
        // Prompt user for the number of coins and tokens
        let coins = prompt("Enter the number of coins you want to add:", "1000000");
        let tokens = prompt("Enter the number of tokens you want to add:", "1000000");

        // Parse the input values to integers
        coins = parseInt(coins, 10);
        tokens = parseInt(tokens, 10);

        // Check if the inputs are valid numbers
        if (isNaN(coins) || isNaN(tokens)) {
            alert("Please enter valid numbers.");
            return;
        }

        // Simulate adding the coins and tokens
        // In a real scenario, you would make an HTTP request to the server to update the user's balance
        console.log(`Adding ${coins} coins and ${tokens} tokens to your account.`);

        // For demonstration purposes, we'll just log the changes
        // In an actual script, you would interact with the Blooket API or modify the page DOM
        alert(`${coins} coins and ${tokens} tokens have been added to your account.`);

        // Update the balance display on the page (this is just an example)
        updateBalanceDisplay(coins, tokens);
    }

    // Function to update the balance display on the page
    function updateBalanceDisplay(coins, tokens) {
        // Find the balance display elements and update them
        let balanceElement = document.querySelector('.balance'); // Update this selector based on actual page structure
        if (balanceElement) {
            balanceElement.textContent = `Coins: ${coins}, Tokens: ${tokens}`;
        }
    }

    // Run the function when the page loads
    addCoinsAndTokens();
})();

**Comment by jfullstackdev** [comment]
you can try raising an issue to those public repos of this user, 

and ask what happened to that specific repo

![image](https://user-images.githubusercontent.com/47092464/181499142-897bb1b2-289d-4e9f-886d-bb681d2eb981.png)

**Reply by donprofaghatise** [reply]
OK- thanks jdevstatic for the support

**Comment by uhtr** [comment]
i can't access blooket hack website
